scripts/view.js

```diff
@@ -1,11 +1,34 @@
-var eventController = {};
+(function(module) {
+  
+  var viewController = {};
 
-eventController.handleNav = function() {
-  $('.navtab').click(function() {
-    $('.navtab-content').hide();
-    which = $(this).attr('data-content');
-    $('#' + which).fadeIn();
-  });
-};
+  viewController.handleNav = function() {
+    $('.navtab').click(function() {
+      $('.navtab-content').hide();
+      which = $(this).attr('data-content');
+      $('#' + which).fadeIn();
+    });
+  };
 
-eventController.handleNav();
+  viewController.renderProjects = function() {
+    $projects = $('.project-carousel');
+    Project.projects.forEach(function(proj) {
+      $projects.append(proj.toHTML());
+    });
+    $projects.slick({
+      dots: true,
+      infinite: true,
+      speed: 300,
+      slidesToShow: 1,
+      adaptiveHeight: true,
+      autoplay: true,
+      autoplaySpeed: 6000,
+    });
+  };
+
```

Comment:
TODO: add something clickable to slideIn a longer project description underneath the figcaption. It would also be good to turn off autoplay on the carousel when this is clicked. Then have a button to revert everything back.